test/index.test.ts

```diff
@@ -68,8 +68,11 @@ test('Should ensure formatting is correct for PublicInputs', async function (ass
 test('Should ensure formatting is correct for extracted PublicInputs', async function (assert) {
   // For each circuit
   testVectors.forEach((testVector, i) => {
+    // Ensure testVector proof exists
+    assert.ok(testVector.proof, `Test vector ${i} is missing a proof, which is required for this test`)
```

Comment:
How can the proof be missing from the test vectors?

Reply:
It is not generated until after a `prove()` call, so if someone wanted to generate the `Proof` object this way, by running specific tests first, it is allowed. Instead of requiring building the matching `Proof` object elsewhere.